src/qoi/src/rayfire_mesh.C

```diff
@@ -119,6 +116,10 @@ namespace GRINS
 
         // add new rayfire elem to the map
         _elem_id_map[prev_elem->id()] = elem;
+
+        // add new rayfire elem ID to parent_child map
+        add_parent(elem->id());
```

Comment:
Although technically valid C++, I prefer the `this->add_parent` syntax to make it clear we're calling a member function. Please adjust all other instances as well.